swift/logging/SwiftLogging.cpp

```diff
@@ -130,7 +135,7 @@ void Log::configure() {
       std::filesystem::path diagFile =
```

Comment:
Pre-existing: Is the `if (diagnostics)` supposed to be nested inside `if (text || binary)` or was that an oversight?

Reply:
good catch, that was indeed a bug! :+1: